libmscore/repeatlist.cpp

```diff
@@ -93,12 +75,74 @@ Measure* Score::searchLabelWithinSectionFirst(const QString& s, Measure* section
 RepeatSegment::RepeatSegment()
       {
       tick       = 0;
-      len        = 0;
       utick      = 0;
       utime      = 0.0;
       timeOffset = 0.0;
       }
 
+RepeatSegment::RepeatSegment(RepeatSegment * const rs, Measure * const fromMeasure, Measure * const untilMeasure)
+      {
+      tick       = 0;
+      utick      = 0;
+      utime      = 0.0;
+      timeOffset = 0.0;
```

Comment:
To avoid repeating code here, C++ allows calling one constructor from another:
`RepeatSegment::RepeatSegment(RepeatSegment * const rs, Measure * const fromMeasure, Measure * const untilMeasure)`
`  : RepeatSegment()`
`   {`
`     ...`